src/serum-governance.yaml

```diff
@@ -12,8 +12,256 @@ publishStatus: Live
+
+      When many stakeholders are involved in a particular blockchain network, governance becomes more complicated. Therefore, public Blockchain networks have a much higher governance risk than private Blockchain networks. Anyone who joins a public blockchain can potentially vote and affect the system in some way.
+
+      The community's input is essential for the development of the Serum ecosystem. Here's how discussions among the Serum community turn into Forum posts and proposals on the DAO voting platform:
+      First, someone in the community has an idea or perspective that they think would be beneficial for the ecosystem. They discuss this with others in the community to get feedback and see if there is general agreement. Once there is enough support for the idea, someone will create a post about it on the Forum. From there, others can comment and provide feedback. The goal is to get as much input and feedback as possible so that the proposal can be improved. Once the proposal is ready, it will be posted on the DAO voting platform for everyone to see. All members of the community can then vote on it. If the proposal gets enough votes, it will be implemented in the Serum ecosystem.
+
     stepItems: []
+  - uuid: 8d77177b-46fd-41ad-9655-8b927c694458
+    name: Evaluation
+    content: |
+    stepItems:
+      - uuid: 3445ae29-2e5e-4b7f-aea7-6ae7a13963ad
+        content: Which of the following is one of the key elements for the success of Serum?
+        answerKeys:
+          - C
+        type: SingleChoice
+        choices:
+          - content: Complicated governance
+            key: A
+          - content: Centralization
+            key: B
+          - content: Decentralization
+            key: C
+          - content: High network fees
+            key: D
+
+      - uuid: 36cdcfcf-0251-493a-a520-8ef39a4b2f0d
+        content: What is the need of governance in any Blockchain project?
+        answerKeys:
+          - A
+          - B
+          - C
+        type: MultipleChoice
+        choices:
+          - content: To manage the complexity of decision-making
+            key: A
+          - content: To make sure that a system has the correct incentives, is able to grow and succeed
+            key: B
+          - content: To develop an ecosystem
+            key: C
+          - content: There is no need of governance
+            key: D
+
+  - uuid: 9084a935-3399-4ed1-9b1f-8d540604ad24
+    name: Rules and Guidelines
+    content: |
+      If you're familiar with the DAO Voting Platform UI, you'll know what a realm, governance, and proposal are. If not, here's a quick guide: 
+      **Realm** — an instance of the DAO 
+      **Governance** — an address governed by the DAO (e.g. a token account or program upgrade authority) 
+      **Proposal** — an action to vote on, e.g., transferring tokens or upgrading a program
```

Comment:
`**Proposal**` is mostly a request document on which the DAO needs to build consensus and that is normally done via voting